37502.6 <c-16#199> Created logger
37502.6 <c-16#199:24> Logger was updated
37502.6 <c-16#199:24> Started intializing cube lib
37502.6 <c-16#199:24> Finished intializing cube lib with worker
37502.7 <c-16#199:24> shareCubes is called
37502.7 <c-16#199:24> Added 0 failed local cubes
37502.7 <c-16#199:24> Pruned 0 local cubes
37502.7 <c-16#199:24> CubeSolverThread entered cube retrieval loop, currently there are 0 cubes
37502.7 <c-16#199:24> returnFailedAndRequestCubes => [43] {}
37502.7 <c-16#199:24> Sent 0 failed cubes to root and requested new cubes
37502.7 <c-16#199:24> Received 4 cubes from root
37502.7 <c-16#199:24> CubeSolverThread entered cube retrieval loop, currently there are 4 cubes
37502.7 <c-16#199:24> CubeSolverThread retrieved 4 cubes
37502.7 <c-16#199:24> Started solving a cube
37502.9 <c-16#199:24> Cube failed
37502.9 <c-16#199:24> Added failed cube
37502.9 <c-16#199:24> Started solving a cube
37503.1 <c-16#199:24> Cube failed
37503.1 <c-16#199:24> Added failed cube
37503.1 <c-16#199:24> Started solving a cube
37620.4 <c-16#199:24> appl_pause was called
37620.4 <c-16#199:24> SUSPEND (0.07s since last cb)
38090.1 <c-16#199:24> appl_interrupt was called
38090.1 <c-16#199:24> RESUME
38090.1 <c-16#199:24> STOP after suspension
38090.1 <c-16#199:24> Solving interrupted
38090.1 <c-16#199:24> Leaving the main loop
38090.1 <c-16#199:24> appl_interrupt was called
38090.1 <c-16#199:24> appl_withdraw was called
38090.1 <c-16#199:24> Started withdraw thread
38090.1 <c-16#199:24> Enter destructor of CubeWorkerInterface
38090.1 <c-16#199:24> Enter destructor of CubeRoot
38090.1 <c-16#199:24> Finished withdraw thread
38151.5 <c-16#199:24> Enter destructor
38151.5 <c-16#199:24> Time working: 117.758
38151.5 <c-16#199:24> Time suspended: 469.711
38151.5 <c-16#199:24> Joined cleanup thread
38151.5 <c-16#199:24> Exit destructor
38151.5 <c-16#199:24> Destructing logger
